src/main/views/macro/support.njk

```diff
@@ -56,6 +56,24 @@
   {% endif %}
 {% endmacro %}
 
+{% macro _christmasHolidaysWidget() %}
```

Comment:
you can simply update the condition and dates in this macro, as the previous one is never going to be displayed. No need for a new widget.
https://github.com/hmcts/cmc-citizen-frontend/blob/abdd2416ab8cc711a13480dbfe56783fed84e11c/src/main/views/macro/support.njk#LL46-L57

Reply:
Updated.